ql/src/java/org/apache/hadoop/hive/ql/exec/MoveTask.java

```diff
@@ -569,7 +569,8 @@ public int execute() {
         }
         releaseLocks(tbd);
       }
-
+      long moveFilesDuration = perfLogger.getDuration(PerfLogger.FILE_MOVES);
```

Comment:
Two questions about this: 

1.: This method already contains a perfLogEnd call: 
```java 
      // if _blob_files_kept is present, use it to move the files. Else fall back to normal case.
      if (moveFilesUsingManifestFile(fs, sourcePath, targetPath)) {
        perfLogger.perfLogEnd("MoveTask", PerfLogger.FILE_MOVES);
        return;
      }
```

Does it stops the perfLogger? If yes, how that change will behave in that case? 

2: perfLogEnd is already called at line 196. Can it affect the output of this getDuration call?

Reply:
1. it doesn't stop, just registers and endTime for the duration of FILE_MOVES
2. yeah, depending on the outcome, the ending time should still be registered anyways